### DISCUSSION TITLE
CefCharp won't works on Windows Server (2012 R2 and 2016)

### DISCUSSION THREAD

**Discussion by schivei** [cefsharp/CefSharp#3920] [General]
- **What version of the product are you using?**
    - 96.0.142

- **What architecture x86 or x64?**
    - x64
	
- **What version of .Net?**
    - .Net 6.0
    
- **On what operating system?**
    - Windows Server 2012 R2 and Windows Server 2016

- **Are you using `WinForms`, `WPF` or `OffScreen`?**
    - WPF
    
- **What steps will reproduce the problem?**
    - Publish my application (`dotnet publish -c Release -f net6.0-windows -r win-x64 --self-contained`). 
    - First, I tried to run without Cef, it's works fine, when I use Cef (WPF or WPF.HNWD) it's broken.
    - [Repo Files](https://gist.github.com/schivei/bff93080ed6d115743993e28ed25fd62)

- **What is the expected output? What do you see instead?**
    - I expect to see a blank window with Cef.

- **Please provide any additional information below.**
    - Windows Event Viewer Logs and debug.log can be see [here](https://gist.github.com/schivei/bff93080ed6d115743993e28ed25fd62)

- **Does this problem also occur in the `CEF` Sample Application**
    - No

**Comment by amaitland** [comment]
Converting to discussion as this is unlikely a bug in CefSharp. 

Have you installed Visual C++ x64?

**Reply by schivei** [reply]
Yes, I have VCC x64 installed on server.

I see, the version of VCC is 2013, I've been updated to latest version and it's run.

Thanks a lot
